src/lib.rs

```diff
@@ -31,7 +31,7 @@
 //! [dependencies]
```

Comment:
A couple lines above this we say we support stable but we don't now!

Reply:
Hmm, looks like we duplicate the README in the crate docs. IMO, it is better not to include installation instructions in the crates docs, just how to use the API. I've updated the PR with that change, what do you think?